dispatcher.go

```diff
@@ -276,6 +294,7 @@ func (d *Dispatcher) Start() error {
 		allStarted []transport.Lifecycle
 	)
 
+	d.log.Info("Starting up.")
```

Comment:
S-expressions!

Seriously though...don't mind either way, we should just be consistent.